testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeAllConnectors.java

```diff
@@ -63,7 +63,6 @@ public void extendEnvironment(Environment.Builder builder)
                         "kafka",
                         "kinesis",
                         "kudu",
-                        "localfile",
```

Comment:
I think you missed removing following files: 
* `testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-all/localfile.properties`
* `testing/trino-server-dev/etc/catalog/localfile.properties`